mlibrary/my_account
# Fixing console.log() Errors

# Overview
This PR fixes two errors were found in the `console.log()`:

```js
GET https://account.lib.umich.edu/favicon.ico 404
```

This error was resolved by creating a `favicon.ico` and adding it to the `public/` directory. Extra icons and `manifest.webmanifest` was added ass best practices, according to [How to Favicon in 2021](https://css-tricks.com/how-to-favicon-in-2021/).

```js
TypeError: event.target.addAttribute is not a function
```
`addAttribute` is not an existing method. Instead, one would have to type `setAttribute`. Since `disabled` does not take a value, it's more legible to use the `disabled` method, and defining a boolean.

## Testing
* Go through the website and make sure nothing is broken.
* Check the console to see if any errors show up (that relate to the site and not the web components).
* Check the browser and see if the `favicon` shows up.
